sdk/python/setup.py

```diff
@@ -113,6 +113,10 @@
     "great_expectations>=0.14.0,<0.15.0"
 ]
 
+GO_REQUIRED = [
+    "cffi==1.15.0",
```

Comment:
Is pinning in, or should we leave the patch version/minor version floating?

Reply:
Sure, keeping patch version floating